app/demo-stm32f4-discovery/app.toml

```diff
@@ -50,7 +50,7 @@ path = "../../drv/stm32fx-rcc"
 name = "drv-stm32fx-rcc"
 features = ["f4"]
 priority = 1
-requires = {flash = 4096, ram = 1024}
+requires = {flash = 8192, ram = 1024}
```

Comment:
This now needs 720 more bytes of text and 884 bytes of rodata. Curse you, powers of two!

(But I haven't determined *why* yet, I am guessing that it is either simply "new compiler generates different code" or "cortex-m@0.7.3 uses more for some reason". We'll see more once I get the whole build going, I guess.

Reply:
See https://github.com/oxidecomputer/hubris/pull/438/commits/fe85db89a51ba62b311a805a1c518ebbd911837f for all of the various size changes.